apps/hannk/interpreter/interval.h

```diff
@@ -154,6 +154,18 @@ class SmallVector {
     }
 };
 
+template<typename T, size_t Capacity>
+inline std::ostream &operator<<(std::ostream &s, const SmallVector<T, Capacity> &v) {
```

Comment:
BTW, I think we should remove this (and std::vector's version) at some point, but this is fine for now.

Reply:
It's extremely useful for debugging.

Reply:
I agree, but I still don't think we should keep it long term. But as I said, for now it's fine.